Cargo.toml

```diff
@@ -25,7 +25,7 @@ libsodium-resolver = ["sodiumoxide", "byteorder"]
 libsodium-accelerated = ["libsodium-resolver", "default-resolver"]
 vector-tests = []
 hfs = []
-pqclean_kyber1024 = ["pqcrypto-kyber", "pqcrypto-traits", "hfs", "default-resolver"]
+oqs_kyber1024 = ["oqs", "hfs", "default-resolver"]
```

Comment:
shall we just call this `kyber1024` or `nist3_kyber1024` so we don't have to rename it the next time an underlying crypto library changes?

Reply:
sounds like a good idea to me